src/code/containers/challenge-container-selector.js

```diff
@@ -66,15 +75,14 @@ class ChallengeContainerSelector extends Component {
 function mapStateToProps (state) {
   return {
     authoring: state.authoring,
-    case: state.case,
-    challenge: state.challenge
+    currentRouteSpec: state.routeSpec
```

Comment:
I can see the point, but since it's simply `routeSpec` everywhere else, there's a consistency argument as well. (Which is _not_ to suggest that you should go back and change it everywhere. 😉  I'm just pointing out the issue for future consideration.)